04.ls/ls.rb

```diff
@@ -47,7 +40,7 @@ def output(filenames)
   end
 end
 
-option_params, arguments_without_options = search_option(ARGV)
-target_path = arguments_without_options[0] || './'
-filenames = get_filenames(target_path, option_params)
+options, file_paths = search_option(ARGV)
```

Comment:
searchよりもparseの方がメソッドの役割に即しているように思います。あとoptionかoptionsかも揃えた方がベターですね。

```suggestion
options, file_paths = parse_options(ARGV)
```

Reply:
searchからparseに変更しました。

optionのその部分漏れてました！そこも修正しました。